controllers/actions.github.com/autoscalingrunnerset_controller.go

```diff
@@ -42,13 +42,10 @@ import (
 )
 
 const (
-	// TODO: Replace with shared image.
-	autoscalingRunnerSetOwnerKey             = ".metadata.controller"
-	LabelKeyRunnerSpecHash                   = "runner-spec-hash"
```

Comment:
Removed TODO and unexported `LabelKeyRunnerSpecHash` since it is only used in `autoscalingrunnerset_controller`